test/gurka/test_route_summary.cc

```diff
@@ -7,7 +7,10 @@
 #include <valhalla/baldr/graphreader.h>
 #include <valhalla/baldr/traffictile.h>
 
+#ifndef _MSC_VER
```

Comment:
Should be `_WIN32` like the rest.

Reply:
is win arm64 be included in `_WIN32` too? I guess they have to bcs of compat?